example/lib/03.control/widgets/code_editor_appbar.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+
+import 'dropdown_selector.dart';
+
+class CodeEditorAppbar extends StatelessWidget implements PreferredSizeWidget {
+  final double height;
+  final List<String?> languages;
+  final List<String?> themes;
+  final String? title;
+  final Function(String?)? onLanguageChanged;
```

Comment:
```suggestion
  final ValueChanged<String> onLanguageChanged;
```

Reply:
I need `Function(String?)` so changed to `ValueChanged<String?>`